src/components/UsersCounter/UsersCounter.js

```diff
@@ -0,0 +1,37 @@
+import { h } from 'preact';
+import Icon from 'react-fontawesome';
+
+const UsersCounter = ({ count, label }) => (
+    <div className="users-counter">
+        <div className="count">
+            { count }
+        </div>
+
+        <div className="label">
+            <Icon name="users" />
+            <span>{ label }</span>
+        </div>
+        <style jsx>{`
+            .users-counter {
+                display: flex;
+                font-size: 2em;
+                align-items: center;
+            }
+
+            .count {
+                font-size: 1.5em;
+                margin-right: 5px;
+            }
+
+            .label {
+                display: flex;
+                flex-direction: column;
+                font-size: 0.6em;
+                align-items: center;
```

Comment:
Shouldn't it be left-aligned?
initial css design:
![image](https://user-images.githubusercontent.com/3236551/33239210-f8b4e76a-d2ad-11e7-9a8d-8da372173e49.png)

_qd-ui:
![image](https://user-images.githubusercontent.com/3236551/33239216-0c666752-d2ae-11e7-9e04-4f027887ee0a.png)